cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/servlet/KafkaCruiseControlServletUtils.java

```diff
@@ -193,7 +193,7 @@ static String urlEncode(String s) throws UnsupportedEncodingException {
    */
   static CruiseControlEndPoint getValidEndpoint(HttpServletRequest request, HttpServletResponse response, KafkaCruiseControlConfig config)
       throws IOException {
-    CruiseControlEndPoint endPoint = endPoint(request);
+    CruiseControlEndPoint endPoint = endPoint(request, config.getString(WebServerConfig.WEBSERVER_API_URLPREFIX_CONFIG));
     if (endPoint == null) {
       String method = request.getMethod();
       String errorMessage = String.format("Unrecognized endpoint in request '%s'%nSupported %s endpoints: %s",
```

Comment:
According to the Java Servlet specification, the config `WebServerConfig.WEBSERVER_API_URLPREFIX_CONFIG` must end with `/*` suffix.
1. The following comment seems incomplete: 

https://github.com/linkedin/cruise-control/blob/5ed774eb5c64f6e57a6a001c1306c520170416c5/config/cruisecontrol.properties#L325-L327

Can we update it to say:
```
# REST API default prefix (dont forget the ending /*)
```

2. We don't have any sanity check to enforce this suffix upon configuration time. Can we add a sanity check for `WEBSERVER_API_URLPREFIX_CONFIG` under `KafkaCruiseControlConfig`?